prototypes/factory.lua

```diff
@@ -34,7 +34,7 @@ local factory_1 = function(suffix, localised_suffix, result_suffix, visible, cou
 	local localised_name = {"entity-name.factory-1" .. localised_suffix}
 	local result_name = "factory-1" .. result_suffix
 	local item_flags
-	if visible then item_flags = {"goes-to-quickbar"} else item_flags = {"hidden"} end
+	if visible then item_flags = {} else item_flags = {"hidden"} end
```

Comment:
I'm wondering whether this is the right replacement but I don't know something else for the mods I own too

Reply:
This is the correct replacement and it is what all the other ported mods are doing, I used them as a reference.